drivers/hmis/app/models/hmis/unit_occupancy.rb

```diff
@@ -0,0 +1,41 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class Hmis::UnitOccupancy < Hmis::HmisBase
+  include ::Hmis::Concerns::HmisArelHelper
+  self.table_name = :hmis_unit_occupancy
+
+  # Unit that is occupied
+  belongs_to :unit, class_name: 'Hmis::Unit'
+  # Client enrollment that is occupying the unit
+  belongs_to :enrollment, class_name: 'Hmis::Hud::Enrollment'
+  # Date range for which the client occupied the unit.
+  has_one :occupancy_period, class_name: 'Hmis::ActiveRange', as: :entity, dependent: :destroy
```

Comment:
I think there should only be ONE active range per unit occupancy. How can we constrain that?